scripts/validate-android-test-env.sh

```diff
@@ -10,7 +10,7 @@
 # Check that Buck is working.
 if [ -z "$(which buck)" ]; then
   echo "You need to install Buck."
-  echo "See https://buckbuild.com/setup/install.htm for instructions."
+  echo "See https://buckbuild.com/setup/install.html for instructions."
```

Comment:
I just thought that redirect might break in future. if you think it's not worth the trouble then sure, let's run with it.